CHANGELOG.md

```diff
@@ -3,7 +3,54 @@
 All notable changes to this project will be documented in this file. This
 project adheres to [Semantic Versioning](http://semver.org/).
 
-## Unreleased
+## 0.70.0
+
+This release contains a mix of features, performance improvements, and bugfixes.
+
+### Optimized read mode for OPA's in-memory store ([#7125](https://github.com/open-policy-agent/opa/pull/7125))
+
+A new optimized read mode has been added to the default in-memory store, where data written to the store is eagerly converted
+to AST values (the data format used during evaluation). This removes the time spent converting raw data values to AST
+during policy evaluation, thereby improving performance.
+
+The memory footprint of the store will increase, as processed AST values generally take up more space in memory than the
+corresponding raw data values, but overall memory usage of OPA might remain more stable over time, as pre-converted data
+is shared across evaluations and isn't recomputed for each evaluation, which can cause spikes in memory usage.
+
+This mode can be enabled for `opa run`, `opa eval`, and `opa bench` by setting the `--optimize-store-for-read-speed` flag.
```

Comment:
I don't suppose we have any extra docs to link to for this one? 🤔

Ideally, we'd present some numbers. But that can be misleading and cause false expectations 😅 I guess this paragraph will have to be sufficient.

Reply:
Let's perhaps link to this? https://www.openpolicyagent.org/docs/v0.70.0/policy-performance/#storage-optimization (when the release is done it no longer 404s 🤞)

Reply:
Added a link to the docs.